tests/unit/htdocs/class/CriteriaTest.php

```
@@ -0,0 +1,1829 @@
+
+        // Calling render() without argument should use $GLOBALS['xoopsDB']
+        $this->assertSame('`uid` = 5', $c->render());
+    }
+
+    // =========================================================================
+    // Criteria — renderLdap
+    // =========================================================================
+
+    public function testRenderLdapSimpleEquality(): void
+    {
+        $c = new Criteria('uid', '1000', '=');
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $c. Configured minimum length is 3. (undefined)

(ShortVariable)